tests/src/test/java/alluxio/underfs/UnderStorageSystemInterfaceIntegrationTest.java

```diff
@@ -143,6 +143,27 @@ public void createOpenLarge() throws IOException {
   }
 
   /**
+   * Tests seek.
+   */
+  @Test
+  public void createOpenSeek() throws IOException {
```

Comment:
can you please add another test case for seeing to a pos before the current pos? since Object storage seek might reopen a new stream in this case, it's good to have this code path covered.